tests/test_notify_me.py

```diff
@@ -1,11 +1,10 @@
-from unittest.mock import patch
-
+from unittest.mock import patch, MagicMock
 from notify_me.notify_me import notify_me
 
-
 class TestNotifyMeSuite:
     @patch('notify_me.notify_me.requests')
     def test_send_message(self, requests_mock):
-        notify_me('message', {'url': '127.0.0.1'})
+        response = notify_me('message', {'url': '127.0.0.1'})
 
+        requests_mock.post.return_value == MagicMock(status_code=200)
```

Comment:
also, call assert before this boolean statement. Otherwise it isn't verifying it.